umap/settings/docker.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding:utf-8 -*-
+"""
+Settings for Docker development
+
+Use this file as a base for your local development settings and copy
+it to umap/settings/local.py. It should not be checked into
+your code repository.
+"""
+import environ
+from umap.settings.base import *   # pylint: disable=W0614,W0401
+
+env = environ.Env()
```

Comment:
I'd suggest to keep in `docker.py` only the specific bits, so I'd move the env sniffing in `base.py`. Appart from the new depss, it does not hurt, and may even save someone's day :)